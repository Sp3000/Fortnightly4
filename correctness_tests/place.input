count
place 0 0 red
count
place 1 1 blue
count
place 0 0 blue
count
place 1 1 red
count
halt