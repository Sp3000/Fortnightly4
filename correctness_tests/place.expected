0
1
2
Error: place 0 0 blue
2
Error: place 1 1 red
2